designer/server/src/models/forms/edit.js

```diff
@@ -0,0 +1,21 @@
+import * as create from '~/src/models/forms/create.js'
+
+/**
+ * @param {string} slug
+ * @param {Partial<FormMetadataInput>} [metadata]
```

Comment:
Looks like we call `organisationViewModel()` after a full metadata response:

```js
const metadata = await forms.get(slug, token)
```

Presume the slug via `metadata.slug` works with ~`FormMetadataInput`~ `FormMetadata`

```suggestion
 * @param {FormMetadata} metadata
```

Reply:
Excellent spot! Not sure why I originally started down that route. Fixed.